IdentityCore/src/broker_operation/response/MSIDBrokerOperationTokenResponse.m

```diff
@@ -56,6 +59,16 @@ - (instancetype)initWithJSONDictionary:(NSDictionary *)json error:(NSError **)er
             _authority = (MSIDAuthority *)[MSIDJsonSerializableFactory createFromJSONDictionary:json classTypeJSONKey:MSID_PROVIDER_TYPE_JSON_KEY assertKindOfClass:MSIDAuthority.class error:error];
             if (!_authority) return nil;
         }
+        else
+        {
+            if (![json msidAssertType:NSString.class ofKey:MSID_BROKER_ADDITIONAL_TOKEN_RESPONSE_JSON_KEY required:NO error:error]) return nil;
+            NSString *tokenResponseJsonString = json[MSID_BROKER_ADDITIONAL_TOKEN_RESPONSE_JSON_KEY];
+            if (tokenResponseJsonString)
+            {
+                NSDictionary *tokenResponseJson = (NSDictionary *)[[MSIDJsonSerializer new] fromJsonString:tokenResponseJsonString ofType:NSDictionary.class context:nil error:nil];
+                _additionalTokenResponse = (MSIDTokenResponse *)[MSIDJsonSerializableFactory createFromJSONDictionary:tokenResponseJson classTypeJSONKey:MSID_PROVIDER_TYPE_JSON_KEY assertKindOfClass:MSIDTokenResponse.class error:nil];
```

Comment:
it will be better if we could log in case any error happens :)

Reply:
It is logged in MSIDJsonSerializableFactory.